components/constellation/constellation.rs

```diff
@@ -1382,8 +1389,8 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
     // page navigation.
     fn handle_script_loaded_url_in_iframe_msg(&mut self, load_info: IFrameLoadInfoWithData) {
         let (load_data, window_size, is_private) = {
-            let old_pipeline = load_info.old_pipeline_id
-                .and_then(|old_pipeline_id| self.pipelines.get(&old_pipeline_id));
+            let old_pipeline = self.frames.get(&load_info.info.frame_id)
+                .and_then(|frame| self.pipelines.get(&frame.pipeline_id));
```

Comment:
Looks like some leftover from an attempt to remove `old_pipeline_id` from the load info.